image/store.go

```diff
@@ -325,18 +326,18 @@ func (is *store) imagesMap(all bool) map[ID]*Image {
 		if !all && len(is.children(id)) > 0 {
 			continue
 		}
-		img, err := is.Get(id)
+		img, err := is.Get(ctx, id)
 		if err != nil {
 			logrus.Errorf("invalid image access: %q, error: %q", id, err)
 			continue
 		}
 		images[id] = img
 	}
-	return images
+	return images, ctx.Err()
 }
 
-func (is *store) Len() int {
+func (is *store) Len(context.Context) (int, error) {
```

Comment:
See my other comments; wondering why we're adding an error return here, if there's never an error to return